buildenv/jenkins/common/build

```diff
@@ -195,26 +196,50 @@ def checkout_pullrequest() {
             if (omr_upstream) {
                 sh "git config remote.origin.url https://github.com/eclipse/omr.git"
             }
-            checkout_pullrequest(OMR_PR)
+            checkout_pullrequest(OMR_PR,'omr')
         }
     }
 }
 
-def checkout_pullrequest(ID) {
+def checkout_pullrequest(ID,repo) {
     // Determine if we are checking out a PR or a branch
+    def JSONFILE = ""
+    def base1 = ""
+	def URL = "https://api.github.com/repos/eclipse/${repo}/pulls/${ID}"
```

Comment:
`eclipse` is the org for openj9 and omr but for the openjdk extensions repos it would be `ibmruntimes`.

Reply:
fixed